### ISSUE TITLE
Missing dep Win32::Clipboard prereq on Cygwin

### ISSUE THREAD

**Issue by tzccinct** [shlomif/Clipboard#7]
http://www.cpantesters.org/cpan/report/215179b8-73c1-1014-ba2f-25bc76d8a230

On version 0.17, the following line was removed from Makefile.PL:
`requires 'Win32::Clipboard' if $^O =~ /mswin|cygwin/i;`

Current dist.ini has the following configuration for MSWin32, but not for Cygwin:
```
[OSPrereqs / MSWin32]
Win32::Clipboard = 0
```

I suppose the following configuration is necessary in dist.ini:
```
[OSPrereqs / cygwin]
Win32::Clipboard = 0
```

**Comment by shlomif** [comment]
Should be fixed in 0.28. please test.

**Comment by tzccinct** [comment]
I have tested 0.28 and confirmed it is no problem. Thank you for your update.
